pkg/testing/ess/serverless_test.go

```diff
@@ -16,6 +16,9 @@ import (
 )
 
 func TestProvisionGetRegions(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*240)
```

Comment:
[Question]

Why 240s instead of 4min?
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*4)
```

Reply:
No idea, copied from another part of this file. I switched everything to minutes for consistency in here.